client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -54,6 +63,32 @@ const PlanComponent = ( {
 	)
 		? JetpackBundleCard
 		: JetpackPlanCard;
+
+	if ( plan.upgradeable ) {
+		// TODO: remove this once we can purchase new plans.
+		// We need this now to make the nudge appear inside the Jetpack Security Bundle card
+		// and make it work as it was Jetpack Security Daily (monthly or annually).
+		const productSlug =
+			plan.productSlug === OPTIONS_JETPACK_SECURITY
+				? PLAN_JETPACK_SECURITY_DAILY
+				: PLAN_JETPACK_SECURITY_DAILY_MONTHLY;
+
+		const upgradeToProductSlug = productSlug && getRealtimeFromDaily( productSlug );
+		const item = upgradeToProductSlug && slugToItem( upgradeToProductSlug );
+		const selectorProductToUpgrade = item && itemToSelectorProduct( item );
+
+		plan.UpgradeNudge = selectorProductToUpgrade ? (
+			<JetpackProductCardUpgradeNudge
+				billingTimeFrame={ durationToText( plan.term ) }
+				currencyCode={ currencyCode }
+				discountedPrice={ 67 }
```

Comment:
Will this be provided by an API eventually?

Reply:
Yes, if the question is if the price is going to come from an API. Now, how the price gets to this specific component is a question I'm still pondering. Do you think the SelectorProduct object could/should contain the prices as well? If yes, we can get the price from the SelectorProduct above.

Reply:
That's the goal eventually, yes! It is going to be part of 1169247016322522-as-1188698065630107. For now, I think we can leave this.